charts/prometheus/values.yaml

```
@@ -371,9 +371,7 @@ server:
   ## PodDisruptionBudget settings
   ## ref: https://kubernetes.io/docs/concepts/workloads/pods/disruptions/
   ##
-  podDisruptionBudget:
-    enabled: false
-    maxUnavailable: 1
+  podDisruptionBudget: {}
```

The field can now include spec as desired (apart from the selector which is already preset in the template).

```suggestion
  podDisruptionBudget:
    enabled: false
    maxUnavailable: 1
    # minAvailable: 1
    ## unhealthyPodEvictionPolicy is available since 1.27.0 (beta)
    ## https://kubernetes.io/docs/tasks/run-application/configure-pdb/#unhealthy-pod-eviction-policy
    # unhealthyPodEvictionPolicy: IfHealthyBudget
```